### PR TITLE
feature/CD-40: refactoring of /create logic

### PR DESCRIPTION
**Pull request by Gerasimov94**

### CODE REVIEW

File: src/modules/CreateImage/components/CreateImageInitializationHeaderComponent/CreateImageInitializationHeaderComponent.tsx [hunk 1]
```diff
@@ -7,36 +7,108 @@ import Flex from 'common/components/Flex/Flex';
 
 import styles from './createImageInitializationHeaderComponent.scss';
 
+import capitalize from 'utils/capitalize';
```
Comment (CodeAnnihilator) [inline src/modules/CreateImage/components/CreateImageInitializationHeaderComponent/CreateImageInitializationHeaderComponent.tsx:10]: reorder, please, according to this guide https://trello.com/c/VcHAzRFZ/6-i-want-us-to-have-a-certain-import-files-order

File: src/modules/CreateImage/components/CreateImageInitializationHeaderComponent/CreateImageInitializationHeaderComponent.tsx [hunk 2]
```diff
@@ -7,36 +7,108 @@ import Flex from 'common/components/Flex/Flex';
...
 					<Flex indent='medium'>
-						<Trans i18nKey='CreateImage.InitializationStep' />
-					</Flex>
-					<Flex indent='medium'>
-						<Alert type='warning'>
-							<Trans i18nKey='CreateImage.DeviceNotSelected' />
-						</Alert>
+						<Trans i18nKey={`CreateImage.${capitalize(step)}Step`} />
 					</Flex>
+					{alertTitleKey
```
Comment (CodeAnnihilator) [inline src/modules/CreateImage/components/CreateImageInitializationHeaderComponent/CreateImageInitializationHeaderComponent.tsx:52]: why not just using && ?

File: src/utils/capitalize.ts [hunk 3]
```diff
@@ -0,0 +1,3 @@
+export default function capitalize(text: string) {
+	return `${text[0].toUpperCase()}${text.slice(1, text.length)}`;
```
Comment (gusartus) [inline src/utils/capitalize.ts]: I guess we can replace .slice with .substr
${text.slice(1, text.length)} --> &{text.substr(1)}

File: src/modules/CreateImage/components/CreateImageInitializationHeaderComponent/CreateImageInitializationHeaderComponent.tsx (hunk 2 above)
Comment (Gerasimov94) [reply src/modules/CreateImage/components/CreateImageInitializationHeaderComponent/CreateImageInitializationHeaderComponent.tsx:52]: i don't like this way

**Review by CodeAnnihilator — APPROVED** [review]

**Pull request merged**